keycloak fails to build on amd64

The package keycloak with version 26.1.1 and epoch r0 fails to build from source with the following error:

```
running step "git-checkout"
[git checkout] repo='https://github.com/keycloak/keycloak' dest='.' depth='1' branch='' tag='26.1.1' expcommit='217a341f3789368d1379185b80b1dd11d7f1e884' recurse='false'
[git checkout] execute: git config --global --add safe.directory /tmp/tmp.I1aSVC
[git checkout] execute: git config --global --add safe.directory /home/build
[git checkout] execute: git clone --quiet --origin=origin --config=user.name=Melange Build --config=user.email=melange-build@cgr.dev --config=advice.detachedHead=false --branch=26.1.1 --depth=1 https://github.com/keycloak/keycloak /tmp/tmp.I1aSVC
[git checkout] execute: cd /tmp/tmp.I1aSVC
[git checkout] tar -c . | tar -C "/home/build" -x
[git checkout] execute: cd /home/build
[git checkout] execute: git config --global --add safe.directory /home/build
[git checkout] execute: git fetch --quiet origin --depth=1 --no-tags +refs/tags/26.1.1:refs/origin/tags/26.1.1
[git checkout] execute: git checkout --quiet origin/tags/26.1.1
[git checkout] tag 26.1.1 is 217a341f3789368d1379185b80b1dd11d7f1e884
running step "patch"
Hmm...  Looks like a unified diff to me...
The text leading up to this was:
--------------------------
|diff --git a/pom.xml b/pom.xml
|index 3404e1d..b324888 100644
|--- a/pom.xml
|+++ b/pom.xml
--------------------------
patching file pom.xml
Using Plan A...
Hunk #1 succeeded at 1293 (offset 5 lines).
done
log4j:WARN No appenders could be found for logger (org.jboss.logging).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
Feb 07, 2025 7:58:05 AM org.infinispan.protostream.annotations.impl.ProtoMessageTypeMetadata scanMemberAnnotations
WARN: Class org.keycloak.models.sessions.infinispan.stream.CollectionToStreamMapper does not have any @ProtoField annotated members. The class should be either annotated or it should have a custom marshaller.
Feb 07, 2025 7:58:05 AM org.infinispan.protostream.annotations.impl.ProtoMessageTypeMetadata scanMemberAnnotations
WARN: Class org.keycloak.models.sessions.infinispan.stream.AuthClientSessionSetMapper does not have any @ProtoField annotated members. The class should be either annotated or it should have a custom marshaller.
Feb 07, 2025 7:58:05 AM org.infinispan.protostream.annotations.impl.ProtoMessageTypeMetadata scanMemberAnnotations
WARN: Class org.keycloak.models.sessions.infinispan.stream.MapEntryToKeyMapper does not have any @ProtoField annotated members. The class should be either annotated or it should have a custom marshaller.
Feb 07, 2025 7:58:05 AM org.infinispan.protostream.annotations.impl.ProtoMessageTypeMetadata scanMemberAnnotations
WARN: Class org.keycloak.models.sessions.infinispan.stream.GroupAndCountCollectorSupplier does not have any @ProtoField annotated members. The class should be either annotated or it should have a custom marshaller.
Feb 07, 2025 7:58:05 AM org.infinispan.protostream.annotations.impl.ProtoMessageTypeMetadata scanMemberAnnotations
WARN: Class org.keycloak.models.cache.infinispan.ClearCacheEvent does not have any @ProtoField annotated members. The class should be either annotated or it should have a custom marshaller.
Feb 07, 2025 7:58:05 AM org.infinispan.protostream.annotations.impl.ProtoMessageTypeMetadata scanMemberAnnotations
WARN: Class org.keycloak.models.sessions.infinispan.stream.SessionUnwrapMapper does not have any @ProtoField annotated members. The class should be either annotated or it should have a custom marshaller.
[ERROR] Failed to execute goal org.infinispan.protostream:proto-schema-compatibility-maven-plugin:5.0.12.Final:proto-schema-compatibility-check (default) on project keycloak-model-infinispan: Execution default of goal org.infinispan.protostream:proto-schema-compatibility-maven-plugin:5.0.12.Final:proto-schema-compatibility-check failed: IPROTO000039: Incompatible schema changes:
[ERROR] IPROTO000036: Field 'keycloak.RoleAddedEvent.roleName' was removed, but its name has not been reserved
[ERROR] IPROTO000037: Field 'keycloak.RoleAddedEvent.roleName' was removed, but its number 3 has not been reserved
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :keycloak-model-infinispan
failed to build package: unable to run package keycloak pipeline: unable to run pipeline: exit status 1
```

--- a/keycloak.yaml
+++ b/keycloak.yaml
@@ -1,7 +1,7 @@
 package:
   name: keycloak
   version: "26.1.2"
-  epoch: 1
+  epoch: 2
   description: Open Source Identity and Access Management For Modern Applications and Services
   copyright:
     - license: Apache-2.0
@@ -61,7 +61,7 @@ pipeline:
       # Gross hack to work around broken NAPI ast-grep module that has
       # undefined symbol: static_assert
       export LD_PRELOAD=/tmp/preload.so
-      ./mvnw clean install -DskipTests=true -Pdistribution -q
+      ./mvnw clean install -DskipTests=true -Pdistribution -q -DskipProtoLock=true
       unset LD_PRELOAD
 
       mkdir -p ${{targets.destdir}}/usr/share/java

